web/saml.go

```diff
@@ -84,8 +84,12 @@ func completeSaml(c *Context, w http.ResponseWriter, r *http.Request) {
 		relayProps = model.MapFromJson(strings.NewReader(stateStr))
 	}
 
+	c.LogAudit("attempt")
```

Comment:
By doing this "attempt" entry here we're getting the following...

```
             id             |   createat    |           userid           |     action      |   extrainfo    | ipaddress | sessionid
----------------------------+---------------+----------------------------+-----------------+----------------+-----------+-----------
 ekeybqxc7b8o7eqhnzrmnec1by | 1571918658800 |                            | /login/sso/saml | attempt        | ::1       |
```

... which tells us that a particular IP address has made a POST request to the /login/sso/saml endpoint at a particular time. I'm just wondering if it would be better to log this inside the `(*SamlInterfaceImpl).DoLogin` method so that we can get additional information from the decrypted XML? Thoughts?

Reply:
See above.

Reply:
Per Dennis comments above, will revisit this once we get a better handling of the Audit feature in general.